src/main/java/pl/piekoszek/gorskimatches/image/ImageController.java

```diff
@@ -20,6 +21,7 @@ class ImageController {
         this.lineImageCreator = lineImageCreator;
     }
     @GetMapping(value = "{number}", produces = MediaType.IMAGE_PNG_VALUE)
+
```

Comment:
and this

Reply:
I tested and when I commented this line the URL: http://localhost:8080/number.html?number=1 wasn't displaying the numbers. Or are you commenting on additional enter?

Reply:
Yep, it was regarding empty line ;)